homework7/db.php

```diff
@@ -0,0 +1,144 @@
+ <?php 
+	$articles = array(
+		array(
```

Comment:
@Niki1993 
This will work, but it will be better if you put the articles in seperate sections. You can do this by either:
1. Move them in separate variables :
2, Create three-dimensional array .

: )